modules/local/validation/metadata/validate_metadata.py

```diff
@@ -0,0 +1,268 @@
+
+
+def load_analysis_payload(payload_file):
+    """Load and validate basic structure of analysis payload JSON."""
+    try:
+        with open(payload_file, 'r') as f:
+            payload = json.load(f)
+        
+        # Check required fields
+        required_fields = ['studyId', 'analysisType', 'files']
+        missing_fields = []
+        
+        for field in required_fields:
+            if field not in payload:
+                missing_fields.append(field)
+        
+        if missing_fields:
+            raise ValueError(f'Missing required fields: {", ".join(missing_fields)}')
+        
+        return payload
+    
+    except FileNotFoundError:
+        raise FileNotFoundError(f'Analysis payload file not found: {payload_file}')
+    except json.JSONDecodeError as e:
+        raise ValueError(f'Invalid JSON in payload file: {e}')
+
+
+def load_read_group_data(read_group_file):
+    """Load read group data from TSV file."""
+    try:
+        read_groups = []
+        with open(read_group_file, 'r') as f:
+            reader = csv.DictReader(f, delimiter='\t')
+            read_groups = list(reader)
+        
+        if not read_groups:
+            raise ValueError('No read groups found in read group file')
+        
+        return read_groups
+    
+    except FileNotFoundError:
+        raise FileNotFoundError(f'Read group file not found: {read_group_file}')
+    except Exception as e:
+        raise ValueError(f'Error reading read group file: {e}')
+
+
+def validate_read_group_files(payload, read_groups):
+    """Validate file references between payload and read groups."""
+    validation_errors = []
+    
+    # Get payload files
+    payload_files = payload.get('files', [])
+    payload_filenames = set()
+    
+    for file_entry in payload_files:
+        if 'fileName' in file_entry:
+            payload_filenames.add(file_entry['fileName'])
+    
+    print(f'Found {len(payload_filenames)} files in payload: {sorted(payload_filenames)}')
+    
+    # Cross-validate read group file references with payload files
+    read_group_files = set()
+    
+    for rg in read_groups:
+        rg_id = rg.get('submitter_read_group_id', 'unknown')
+        
+        # Check file_r1
+        file_r1 = rg.get('file_r1', '').strip()
+        if file_r1:
+            read_group_files.add(file_r1)
+            if file_r1 not in payload_filenames:
+                validation_errors.append(f'Read group {rg_id}: file_r1 "{file_r1}" not found in payload files')
+        
+        # Check file_r2
+        file_r2 = rg.get('file_r2', '').strip()
+        if file_r2:
+            read_group_files.add(file_r2)
+            if file_r2 not in payload_filenames:
+                validation_errors.append(f'Read group {rg_id}: file_r2 "{file_r2}" not found in payload files')
+        
+        # Validate library layout consistency
+        library_layout = rg.get('library_layout', '').upper()
+        if library_layout == 'PAIRED':
```

Comment:
Accepted lectern value is [OBI:0000722 (paired-end library)](https://github.com/Pan-Canadian-Genome-Library/data-dictionary/blob/c6266223e2b594fa514a1d56675e56cbd865b1c2/lectern/pcgl/pcgl_lectern.json#L2722)